asm/single_precision

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+echo Generating dsdot.go
+echo '// Generated code do not edit. Run `go generate`.\n' > dsdot.go
```

Comment:
Sorry, just found out after running on my system: you need to remove `\n` and add an empty echo command afterwards. My echo command doesn't evaluate `\n`.